cmd/kubeadm/app/phases/certs/pkiutil/pki_helpers.go

```diff
@@ -34,15 +34,12 @@ import (
 )
 
 // NewCertificateAuthority creates new certificate and private key for the certificate authority
-func NewCertificateAuthority() (*x509.Certificate, *rsa.PrivateKey, error) {
+func NewCertificateAuthority(config certutil.Config) (*x509.Certificate, *rsa.PrivateKey, error) {
```

Comment:
`config` should be a struct pointer unless there is a reason for `config` to be a copy

Reply:
yeah I thought that was weird too, but was erring on the side of Chesterton's face. I'll fix it.